app/views/posts/index.html.haml

```diff
@@ -0,0 +1,32 @@
+%h1
+  =@board.title
+#post_table
+  = render @posts
+
+%button.btn.btn-primary.btn-lg{"data-target" => "#NewPost", "data-toggle" => "modal", :type => "button"}
+  Nowy post
+
+=link_to "Nowy post", new_board_post_path(@board), class:"btn btn-primary"
+=link_to "Back", :back, class:"btn btn-info"
+
+=form_for [@board,@post], remote: true do |f|
+  #NewPost.modal.fade{"aria-labelledby" => "myModalLabel", :role => "dialog", :tabindex => "-1"}
```

Comment:
Generalnie nie używamy camelCase, tylko snake_case.
Ogólnie ja przyjąłem metodę że w css-ach klasy-piszemy-tak, a_idki_w_ten_sposób, i to się sprawdza.